stable/kommander/values.yaml

```diff
@@ -233,13 +233,16 @@ kubecost:
 
     fullnameOverride: "kommander-kubecost-cost-analyzer"
     global:
+      # We need to specify full domain name explicity in order to make
+      # http_proxy work.  Otherwise requests go through proxy instead directly
+      zone: cluster.local
```

Comment:
What happens if a customer uses a different cluster domain?

Reply:
They can specify it in the override. The idea is that A domain needs to be specified to make charts use absolute domain names.

Reply:
Can we please document how to set the cluster domain name?

Reply:
Yes, but this is out of the scope of this Jira/PR I think.

Reply:
>  Yes, but this is out of the scope of this Jira/PR I think.

I'm kind of on the fence about this. Before this change this component was agnostic to the cluster domain. Now it is not so we should document that fact right away so that users are able to apply the correct configuration in case they use a different cluster domain.

Reply:
These are defaults. If the customer was using custom domain before, they most probably were using an override of some sort, so nothing changes for them. The users this change addresses are the ones that did not provide any domain. For them, the chart was falling back to non-absolute dns names which break with http_proxy.

Reply:
Responded in https://mesosphere.slack.com/archives/C01LNF6KML5/p1637688810412800